pkg/registry/apis/provisioning/register.go

```diff
@@ -320,7 +330,7 @@ func (b *ProvisioningAPIBuilder) beginUpdate(ctx context.Context, obj, old runti
 
 	undo, err := repo.BeginUpdate(ctx, b.logger, oldRepo)
 	if err != nil {
-		return nil, err
+		b.logger.Warn("error in begin update", "err", err)
```

Comment:
^^ this is required so we can save invalid fils

Reply:
Now that we have a controller -- we should look at how/if we can remove the Begin/After logic